\chapter{Conclusions and perspectives}
The space of known viral biodiversity is increasing at such a pace that the official viral taxonomy structure struggles to keep up \cite{Adriaenssens2020-gb}, yet the degree of viral genomic diversity and variation between biotic environments is suggesting that only a fraction of viral diversity have been identified \cite{Dance_undated-vk}. A great proportion of the established human gut viruses originate from the first series of metagenomic studies using viral-enrichment strategies that selects for a limited subset of the virome \cite{Norman2015-eb,Clooney2019-nn,Manrique2016-vq,Gogokhia2019-li,Shkoporov2019-mk}. The viral-enrichment strategy has been imperative to face the technical challenges involved in virus assembly and identification from metagenomics due to the wealth of genetic remnants from other biological organisms, but also impose restrictions on the type of viral diversity studied \cite{Roux2019-dc,Gregory2020-gu}.  The costs and non-trivial implementation of \textit{in vitro} viral enrichment is a strong motivator for alternative strategies to identify viruses in the growing number of metagenomic samples produced to study biodiversity in biotic and abiotic environments \cite{Wooley2010-mr}.\\ 

\noindent
In this thesis, I have presented computational methods based on deep-learning frameworks that improve the recovery of both prokaryotic, viral and potentially other MGE genomes from bulk metagenomics \textbf{(Paper I and Paper II)}. Importantly, these methods can be applied across metagenomes collected from different environments and allow investigations into dominant hubs of viruses and bacteria in disease and co-evolutionary dynamics of bacteria and viruses. Our study on the gut microbiomes of people with extreme longevity illustrate an important strength of these methods as they can be applied to various metagenomic cohorts and facilitate combined bacterial and viral analysis to answer biological questions such as the human age-dependent impact on ecological viral communities \textbf{(Paper III)}. It is worth noting that the viromes characterized from bulk metagenomics without viral enrichment does not seem to capture the entirety of virome diversity and may be biased towards viruses infecting dominant host cells. RNA-viruses can be abundant in the human gut during disease \cite{Zuo2021-ll}, but their discovery is dependent on metatranscriptomics and construction of cDNA libraries \cite{Callanan2020-xy}. Identification of Microviridae viruses might also be better captured with viral-enrichment, which however could be biased toward micro viruses and virulent viruses but miss larger bacteriophages and integrated proviruses \cite{Roux2019-dc,Parras-Molto2018-wy,Gregory2020-gu}. Ideally, the microbiome should be studied using a combination of both approaches to capture the best picture of the entire virome simultaneously with studying larger organisms like bacteria. However, viral enrichment adds additional costs to a study with focus on the entire microbiome community as a result of further preparation and sequencing expenses. Therefore, a less costly compromise is a greater focus on maximizing virus discovery from bulk metagenomics, which has also been suggested to yield a comparable number of viral contigs to VLP preparations \cite{Gregory2020-gu}. The extent to which virus genome quality and discovery in metagenomes can be improved using long-read technologies is an interesting topic which deserves more attention. Especially since long-read technologies have become a more cost-effective approach to study prokaryotic biodiversity in metagenomics \cite{Moss2020-ot,Sereika2022-ii}.\\

\noindent
For future virome analysis in bulk metagenomes we propose a combined short and long-read sequencing approach to improve assembly and binning of bacterial MAGs (including integrated proviruses), viral MAGs and MGEs. In terms of the functional influence of viruses in an ecological space, there is a dire need for new computational models to explore the unannotated viral genomes. Fortunately, there is an increased adoption of deep language models on protein sequences \cite{Bepler2021-xn}, which could help accelerate the annotation process of the growing bulk of virus protein-coding genes. Computational models for ab initio structure modeling of virus proteins are available \cite{Jumper2021-dd}. In addition, deep learning language models can be used to distill informative statistical embeddings of unannotated virus sequences which can be connected to functionally annotated proteins \cite{Alley2019-el}. Altogether, improved annotation of virus gene-content should increase our understanding of the virus influence on bacterial constituents through predation mechanisms or by contribution of auxiliary metabolic genes in a provirus or episomal state, which have profound implications on the environment and host \cite{Thompson2011-oc,Mayneris-Perxachs2022-wi}. In addition, there might be many complex mechanisms of bacterial and viral teamwork not discovered yet, such as how gut \textit{Bacteroides spp.} benefit from proviruses that induce the release of inosine \cite{Brown2021-dw}. Our understanding of the human gut virome and its interplay with bacterial constituents is still in its infancy \cite{Garmaeva2019-ox}, but recent and new computational methods and databases will help to fuel future discoveries in metagenomic datasets.