New advances in methods and analysis are needed to address the impact on macroecology by the thousands of viruses present in biotic environments such as the human gut \cite{Carlson2019-lm}. The gut microbiota is tightly connected to human health and so far has been a major focus of research initiatives such as the American Human Microbiome Project (HMP)\cite{Turnbaugh2007-nw} and the European MetaHIT project \cite{Ehrlich2011-el}. There is a great desire to expand the knowledge sphere of gut ecology to less characterized segments of the gut community such as the viral kingdom. Bacterial infecting viruses (bacteriophages) are suggested to impact bacterial density and diversity, thus filing a profound niche in the environment. Gut viruses have largely been characterized in multiple studies using viral enrichment methods (Clooney et al. 2019; Shkoporov et al. 2019; Norman et al. 2015; Roux et al. 2016). This procedure greatly improves the metagenomic assembly and identification of gut viruses but also biases the types of virus studied by capturing a limited segment of virome diversity (Roux, Adriaenssens, et al. 2019; Gregory et al. 2020). Hence, improved methods for mining viral biodiversity in bulk metagenomic samples are needed to enable virome analysis without viral-enrichment and uncover the full spectrum of virome diversity in future metagenomic datasets.\\
\noindent
Towards facilitating virome analysis on the growing number of metagenomic samples and enabling exploration into bacterial and viral communities in biotic sites like the human gut, I present and discuss key results of three major studies that have worked toward this aim. First, the metagenomic binning engine VAMB that has provided a fast and reliable framework for genome reconstruction. Second, our exploration and benchmark of viruses extracted from bulk metagenomics and paired metaviromes. Third, an application of our methods to delineate novel viral diversity in humans of extreme longevity and an analysis on the age-dependent impact on viral and bacterial interactions.

\section{Project I: Deep learning for binning and high resolution taxonomic profiling of microbial genomes}

Discovery of novel gut microbiome residents has been accelerated with computational methods such as metagenomic binning, which organize metagenomic assembled DNA sequences, corresponding to chromosome fragments, from the same organism into genome-bins \cite{Almeida2019-fk}. Several attempts have been made to reconstruct thousands of microbial species from massive metagenomics datasets of hundreds of people \cite{Parks2017-jk,Pasolli2019-ik}, by independently assembling and binning each metagenomic sample into genomes. Single-sample binning allows massive parallel processing of samples but does not leverage co-abundance information across samples. Other methods that are developed to perform binning using co-abundance information from all samples deduplicate sequences before binning \cite{Kang2019-su,Wu2016-fe}, which may mask strain-level genome variation and produce intersample chimeric genomes. These chimeras do not represent real microbial genomes and it would be preferable to have such strains assembled per sample and enable functional strain comparison. The main difference between VAMB and existing binners including MetaBAT2, MaxBin2, Canopy and others is that VAMB utilizes unsupervised deep learning to encode contigs into lower-dimensional latent embeddings based on integrated information of co-abundance and sequence composition structure. In order to test VAMB’s performance in reconstructing bacterial genomes, it was applied to (1) established simulated datasets for metagenomic binning benchmarks \cite{Sczyrba2017-ay} and (2) a real metagenomic dataset comprising 1000 metagenomes \cite{Almeida2019-fk}.

\section{Project II: Genome binning of viral entities from bulk metagenomics data}

In the second project we explored genome binning of virus constituents in metagenomics using VAMB as our binning engine. One key feature of the method is, besides state-of-the-art binning performance, it learns to group genomes from the same organisms across samples. In other words, across a metagenomics dataset it learns which genomes are from the same species. We therefore hypothesized that besides bacterial genomes it could also bin and learn viral species despite their astounding diversity \cite{Aggarwala2017-nz}. This would provide an important advancement to cataloging viral species that are notoriously difficult to separate due to the lack of conserved taxonomic markers 
\cite{Roux2019-dc}. Specifically, if the autoencoder framework effectively separates bacterial species on strain-level based on abundance and sequence composition, can it do the same for viruses? To evaluate VAMB’s ability to capture individual viruses as bins, we had access to the largest dataset of deep-sequenced paired metagenome and metavirome samples from the human gut. This dataset encompassed 662 metagenomic and 662 metavirome paired samples obtained from infants at 1 year of age in the Danish COPSAC cohort \cite{Shah_undated-vc}.\\

\noindent
Viruses from the metaviromes were assembled, quality-controlled and de-replicated to establish a ground truth set of viruses. With a set of labeled viruses, we looked up the origin of each sequence in a putative viral bin generated with assembly and binning of the paired bulk metagenomic samples, thus establishing whether a bin corresponded to a real virus. This enabled us to compute degrees of recall/completeness and contamination of viral bins, i.e. does every sequence in a viral bin map to the nearest reference virus in the truth set and does the bin correspond to a complete virus. From these calculations we established the completeness of viral bin recovered in bulk metagenomics and the viral overlap with viruses assembled from metaviromes based on viral enrichment. These efforts provided a list of annotated metagenomic viral bins that we leveraged for training a supervised viral prediction for bin classification in metagenomics. To create a training and validation set, viral bins were combined with bacterial bins corresponding to bacterial metagenome assembled genomes (MAGs). Key genomic features recorded for each bin such as bacterial and viral marker genes were used to train a Random Forest (RF) model to distinguish the two types of microbiome constituents. The RF model performance was evaluated on annotated metagenomic bins derived from the processing of the Diabimmune dataset containing 112 paired metagenomic and 112 metavirome samples \cite{Zhao2017-uf}. To support the RF model performance on real-datasets, the model was further evaluated on simulated datasets containing virus, plasmids and bacteria generated with tools by the Critical Assessment of Metagenome Interpretation (CAMI) consortium \cite{Fritz2019-ej}.\\
\noindent
Finally we applied our viral binning workflow Phages from metagenomics binning (PHAMB) to a massive public metagenomic dataset, the Human Microbiome Project 2 (HMP2) with IBD cases and controls longitudinally sampled \cite{Lloyd-Price2019-cw}, from which no virome characterisation had been described before. Virus populations derived from this dataset were used to establish longitudinal virome profiles, alpha and beta diversity estimates, separation of samples based on clinical dysbiosis scores and individual phage-dysbiosis associations.

\section{Project III: Centenarians have a diverse population of gut bacteriophages that may promote healthy lifespan}

In the third project we applied our combined VAMB and PHAMB approach to uncover and investigate the bacterial microbiome and virome in centenarians. We investigated Japanese centenarians studied in collaboration with the Broad Institute, Boston, US and the Centre for Supercentenarian Medical Research, Keio University, Japan. Centenarians (Age $>$ 100) and in particular supercentenarians (Age $>$ 110) are examples of humans with exceptional longevity. Studies on centenarians have characterized their unique physiology with a low cardiometabolic risk, preferable lipid profiles and protective plasma biomarkers \cite{Hirata2020-za,Barzilai2004-fd}. In addition, centenarians show great resistance to aging-related diseases. One of the suggested components to contribute to their longevity is the gut microbiome \cite{Wilmanski2021-ov}. An initial characterisation of centenarian microbiomes revealed enrichment of bacteria capable of producing novel secondary bile acids with antibiotic properties towards typical gut pathogens \cite{Sato2021-zh}. Altogether this suggested that centenarians likely exhibit greater resistance towards infectious diseases. Further bacterial and metabolomic analysis of centenarian microbiomes are needed to reveal other host-health related factors in the microbiome.\\

\noindent
The cohort investigated by Sato et al. (2021), consisted of centenarians [n $=$ 176 (172 individuals)], elderly (n $=$ 133), young (n $=$ 61) and represented by far the largest microbiome centenarian dataset published. As the gut virome of centenarians have not been described before, we delineated the virome by combining viral-binning and provirus search in bacterial MAGs. To establish the degree of novel viral diversity, we performed viral clustering of the discovered viral-bins and proviruses into viral operational taxonomic units (vOTUs) with the MGV database, which is the most representative DNA virus and phage database published \cite{Nayfach2021-tq}. In order to place the newly identified vOTUs in the context of known diversity, specific viral protein markers were identified in vOTUs and representative MGV genomes to build a phylogenetic viral tree for identifying branches and clades of novel viruses enriched in centenarians. The bacterial affiliation of viruses were determined by CRISPR-spacers, evidence of integration in bacteria and clustering with proviruses of isolated bacteria.\\

\noindent
The way in which the virome interacts with the bacterial community has so far been studied in infants from birth up until 2 years of age, where the virome undergoes dramatic changes as the pioneering bacteria settle in the gut \cite{Liang2020-lr,Gregory2020-gu}. In order to provide new insights into how the virome interacts with bacteria during the last stage of the human lifespan, we calculated viral-bacterial ratios of temperate viruses from young to centenarian microbiomes. Because we had developed a framework to establish the virome in bulk metagenomics, we could include two additional cohorts (infant and another young cohort) in the analysis to establish that the calculated viral-bacterial ratios (VBRs) were reliable estimators of lysogenic activity between groups. We hypothesized that if the calculated VBR distributions captured overall trends or differences of lysogeny in the microbiome, we could compare general viral-bacterial interactions for different age-groups regardless of bacterial community composition. This analysis was limited to confidently annotated temperate viruses as these are capable of switching between a lytic and lysogenic lifestyle. Finally, as viruses are known to influence bacterial metabolism by infection \cite{Mayneris-Perxachs2022-wi}, we characterized and investigated viral genes in search for auxiliary metabolic genes (AMG) related to metabolic systems and pathways in host-bacteria.

\newpage
\section{Datasets overview}

The three projects featured in this dissertation are based on a wealth of different datasets. Here I provide an concise overview and description of each. In the overview I refer to bulk metagenomic samples from the human gut to human gut microbiomes. In addition, I refer to viral-enriched metagenomic samples as human viral metagenomes.

\begin{enumerate}[label=Dataset \Roman*.]
    \item Almeida \cite{Almeida2019-fk}. A cross sectional study of 11,850 human gut microbiomes from 75 different studies. From this study we sampled 1,000 metagenomic samples.
    
    \item CAMI datasets \cite{Sczyrba2017-ay}. Simulated metagenomic benchmark datasets.
    
    \item COPSAC 2010 \cite{Shah_undated-vc}. A cross sectional study of 647 healthy Danish infants. The dataset includes 647 paired human gut microbiomes and viral metagenomes. 
    
    \item Diabimmune Type 1 Diabetes (T1D) \cite{Zhao2017-uf}. Longitudinal study of 33 infants genetically predisposed to T1D. The dataset includes 220 paired human gut microbiomes and human viral metagenomes.
    
    \item Human microbiome project 2 (HMP2) IBD \cite{Lloyd-Price2019-cw}. Longitudinal study of 132 of participants with Crohn's disease (CD), Ulcerative colitis (UC) or no IBD (nonIBD). The dataset comprises 1337 human gut microbiomes.
    
    \item The Japanese centenarian cohort \cite{Sato2021-zh}. Cross sectional study of Japanese adults of three different age categories. The dataset comprises human gut microbiomes from 176 centenarians ($>$100 years old), 110 elderly ($<$100 years old) and 44 young ($>$18 and $<$55 years).
    
    \item The Sardinian centenarian cohort \cite{Wu2019-wo}. Cross sectional study of Sardinian adults of three different age categories. The dataset comprises human gut microbiomes from 19 centenarians ($>$100 years old), 23 elderly ($<$100 years old) and 17 young ($>$18 and $<$55 years).
    
    \item EDIA cohort \cite{Vatanen2022-vg}. Longitudinal study of 142 infants and mothers from Finland, which were followed across the first year of the child's life. From this dataset we selected 668 bulk microbiomes of infants.
    
    \item Tanzania 300FG \cite{Strazar2021-fu}. Cross sectional study of 315 adults from Tanzania. The dataset comprises 315 human gut microbiomes.

\end{enumerate}